enclave_wrangler/objects_api.py

```diff
@@ -354,22 +352,6 @@ def fetch_cset_and_member_objects(
     flat_expression_items = []
     flat_member_items = []
     for cset in cset_versions:
-
-        # TODO: @joeflack4, need an audit log of successful as well as failed fetches so that we can make a quicker
```

Comment:
@Sigfried RE:
> TODO: @joeflack4, need an audit log of successful as well as failed fetches so that we can make a quicker version of derived tables update

I decided not to implement _logs of successful fetches_ because I don't see how it helps with updating the derived tables:
1. We don't need to log successful fetches in order to improve derived tables (from completely remaking them to just doing inserts). `fetch_cset_and_member_objects()` returns the data and then uses it to insert directly into the core tables. So all that needs to be done is to do the same thing for the derived tables.
2. If we fetch data, log successes, but then an error occurs, the logged successes will contain any data needed to insert data anyway. We'll need to fetch the data again.

Maybe there is some other reason to log successful fetches, but I'm not clear on it. 

Is it not more important that we log _successful imports_ rather than successful fetches? That is, we should want some sort of logging done at the end of the refresh, rather than at the end of `fetch_cset_and_member_objects()`?

If I can get some clarification on of we should indeed log successful imports rather than fetches, and what that will be useful for (if only just to have some record), I will implement.